### PR TITLE
Feat/add modal

### PR DESCRIPTION
**Pull request by alejandroGonGon**

### CODE REVIEW

File: modal/_modal.tsx [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+
+interface Props{
+    isVisible: boolean,
+}
+export const ModalContent: React.FC<Props> = ({isVisible}) =>{
+   
+    return(
+        
+    )
+}
```
Comment (nicocadq) [inline modal/_modal.tsx]: not needed file, delete it

File: pages/index.tsx [hunk 2]
```diff
@@ -11,11 +11,16 @@ import { getRandomIcon } from "utils/get-random-icon";
 import { persistEmoji, getStoredEmoji } from "utils/storage";
 
 import styles from "styles/Home.module.css";
+import Link from "next/link";
```
Comment (nicocadq) [inline pages/index.tsx]: this import should be with the third-party ones

File: pages/index.tsx [hunk 3]
```diff
@@ -11,11 +11,16 @@ import { getRandomIcon } from "utils/get-random-icon";
...
 
 const Home: NextPage = () => {
   const { setTheme } = useNextTheme();
   const { isDark } = useTheme();
   const [currentIcon, setCurrentIcon] = useState<string>("");
+  const [visible, setVisible] = useState(false);
+  const closeHandler = () => {
+    setVisible(false);
+  };
```
Comment (nicocadq) [inline pages/index.tsx]:
```suggestion
  const [isModalVisible, setIsModalVisible] = useState(false);
  
  const onModalClose = useCallback(() => {
    setVisible(false);
  }, []);
```

File: styles/Home.module.css [hunk 4]
```diff
@@ -26,11 +37,12 @@
 
 .footer {
   display: flex;
+  justify-content: center;
+  align-items: center;
```
Comment (nicocadq) [inline styles/Home.module.css]: place-items:center;

**Review by nicocadq — CHANGES_REQUESTED** [review]

File: modal/_modal.tsx (hunk 1 above)
Comment (alejandroGonGon) [reply modal/_modal.tsx]: Done

File: pages/index.tsx (hunk 2 above)
Comment (alejandroGonGon) [reply pages/index.tsx]: Done

File: styles/Home.module.css (hunk 4 above)
Comment (alejandroGonGon) [reply styles/Home.module.css]: Done

**Pull request merged**